js-miniapp-bridge/src/bridge.js

```diff
@@ -93,6 +93,7 @@ MiniAppBridge.getUniqueId = function() {
   return new Promise(function(resolve, reject) {
     return MiniAppBridge.exec(
       "getUniqueId",
+      null,
```

Comment:
Should this be `null` or an empty placeholder object, being explicit in the usage?

Reply:
I guess null is rational which is the representation of no value/request